app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
not bad solution but try to make it in one line. You can use an already existing method that counts the appearance of a char in string, for example :)